app/(gcforms)/[locale]/(form filler)/id/[...props]/lib/tests/notifications.vitsts.ts

```diff
@@ -0,0 +1,23 @@
+import { NotificationsInterval } from "@gcforms/types";
```

Comment:
I believe there's a file typo in the name.  `vitsts.ts` should be `vitest.ts` to be run during the test suite.